### DISCUSSION TITLE
AGH doesn't block ads in some applications

### DISCUSSION THREAD

**Discussion by amrmzr** [AdguardTeam/AdGuardHome#6775] [General]
I have latest AGH version install on raspbian os on pi4 model B with ipv6 off.

Any ideas??

**Comment by whyisthisbroken** [comment]
AGH blocking on DNS Level, some apps use the same address (like reddit) to show you ads.
So AGH can't do nothing against this, it's a different way.
Doesn't matter if ipv6 of or on.

**Reply by amrmzr** [reply]
thanks !!
